README.md

```diff
@@ -1,78 +1,120 @@
-* [Detection Metrics as ROS Node](https://jderobot.github.io/DetectionMetrics/functionality/ros_node/): use Detection Metrics as a ROS Node.
-+ [Labelling](https://jderobot.github.io/DetectionMetrics/resources/gsoc_19/): add or modify labels in the datasets in runtime when running Deployer.
-
-The idea is to offer a generic infrastructure to evaluate object detection models against a dataset and compute the common statistics:
-* mAP
-* mAR
-* Mean inference time.
-
-# What's supported in Detection Metrics.
-
-| Support | Detail                                                  |
-| ------ | ------------------------------------------------------------ |
-| Supported OS  | Multiplatform using Docker                 |
-| Supported datasets  | <ul><li>COCO</li><li>ImageNet</li><li>Pascal VOC</li><li>Jderobot recorder logs</li><li>Princeton RGB dataset</li><li>Spinello dataset</li><li>Open images dataset</li></ul> |                |
-| Supported frameworks   | <ul><li>TensorFlow</li><li>Keras</li><li>PyTorch</li><li>Yolo-OpenCV</li><li>Caffe</li><li>Background substraction</li></ul>  |
-| Supported inputs in Deployer   | <ul><li>WebCamera/USB Camera</li><li>Videos</li><li>Streams from ROS</li><li>Streams from ICE</li><li>JdeRobot Recorder Logs</li></ul> |
 
+# DetectionMetrics
+
+### Project webpage [here](https://jderobot.github.io/DetectionMetrics)
+
+*DetectionMetrics* is a family of toolkits designed to unify and streamline the evaluation of perception models across different frameworks and datasets. With the release of ***DetectionMetrics v1*** we introduced a versatile suite of tools focused on object detection, supporting cross-framework evaluation and analysis. [Cite our work](#cite) if you use it in your research!
+
+<table style='font-size:100%'>
+  <tr>
+    <th><i>DetectionMetrics v1</i></th>
+    <th>&#128187; <a href="https://github.com/JdeRobot/DetectionMetrics/releases/tag/v1.0.0">Code</a></th>
+    <th>&#128214; <a href="/v1">Docs</a></th>
+    <th>&#128011; <a href="https://hub.docker.com/r/jderobot/detection-metrics">Docker</a></th>
+    <th>&#128240; <a href="https://www.mdpi.com/1424-8220/22/12/4575">Paper</a></th>
+  </tr>
+</table>
+
+Now, we're excited to introduce ***DetectionMetrics v2***! While retaining the flexibility of our previous release, *DetectionMetrics v2* is designed with an expanded focus on image segmentation, with plans to extend support to object detection and LiDAR applications. As we move forward, *v2* will be the actively maintained version, featuring continued updates and enhancements to keep pace with evolving AI and computer vision technologies.
+
+<table style='font-size:100%'>
+  <tr>
+    <th><i>DetectionMetrics v2</i></th>
+    <th>&#128187; <a href="https://github.com/JdeRobot/DetectionMetrics">Code</a></th>
+    <th>&#128295; <a href="/v2/installation">Installation</a></th>
```

Comment:
I guess all this reference would work once we merge it

Reply:
my bad, fixed :sweat_smile: